server/reports/attendance/attendance-report.js

```diff
@@ -1,57 +0,0 @@
-import J2C from "json2csv";
```

Comment:
Thanks for renaming this attendance-report  file to ...-endpoint. That was the right name for the commission report equivalents, so it's nice that this is consistent.

Reply:
Yeah I noticed you do that for the commision endpoint so it would only make sense to keep these both consistent. We still should talk about making some other things consistent between our files like error messages, dealing with null parameters in the URL, etc.